### PR TITLE
Fix camera being left on after leaving QR page

### LINKED ISSUE #620: Camera is left on after navigating away from QR scan page
Here is an example:
After clicking on the Scan QR code button on the QR Fight page and navigating back, the camera is left enabled (indicated on the top left of the screen)

image

### PR DESCRIPTION
**Pull request by albi005**
Fixes #620

### CODE REVIEW

File: frontend/src/common-components/QrReader.tsx [hunk 1]
```diff
@@ -8,27 +8,34 @@ interface QrReaderProps {
...
+    let controls: IScannerControls | null
+    let cancelled = false
+
+    async function decodeContinuously() {
+      if (!videoElement.current) return
+      const videoInputDevices = await BrowserCodeReader.listVideoInputDevices()
+      const selectedDeviceId = videoInputDevices[0].deviceId
+      if (cancelled) return
+      controls = await codeReader.current.decodeFromVideoDevice(selectedDeviceId, videoElement.current, (result, err, controls) => {
+        console.debug(result, err)
```
Comment (Isti01) [inline frontend/src/common-components/QrReader.tsx:22]: You left a logging statement here

Comment (Isti01) [reply frontend/src/common-components/QrReader.tsx:22]: Oh that was already there, maybe it's good to remove it

Comment (albi005) [reply frontend/src/common-components/QrReader.tsx:22]: It was already there

**Review by Isti01 — APPROVED** [review]

**Comment by Isti01** [comment]
Pls fix the compile errors

**Comment by albi005** [comment]
> Pls fix the compile errors

I can't see the errors because I don't have access to the vercel things

**Pull request merged**